versions/3.1.0.md

```diff
@@ -2276,64 +2276,64 @@ This object is an extended subset of the [JSON Schema Specification Wright Draft
-- additionalProperties - Value can be boolean or object. Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema. Consistent with JSON Schema, `additionalProperties` defaults to `true`.
+- propertyNames
+- contains
+- const
+- examples
+- if / then / else
+- dependentRequired / dependentSchemas
+- deprecated
+- additionalItems / additionalProperties
+- unevaluatedItems / unevaluatedProperties
+- max Contains / minContains
+- readOnly / writeOnly
+
+The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification.
+
+- exclsuiveMinimum - Value can be a numeric value just like JSON Schema, or for backwards compatibility with older drafts it can accept a boolean to act as a modifier to the `minimum` keyword. The numeric value usage is recommended.
+- exclsuiveMaximum - Value can be a numeric value just like JSON Schema, or for backwards compatibility with older drafts it can accept a boolean to act as a modifier to the `minimum` keyword. The numeric value usage is recommended.
 - description - [CommonMark syntax](http://spec.commonmark.org/) MAY be used for rich text representation.
 - format - See [Data Type Formats](#dataTypeFormat) for further details. While relying on JSON Schema's defined formats, the OAS offers a few additional predefined formats.
-- default - The default value represents what would be assumed by the consumer of the input as the value of the schema if one is not provided. Unlike JSON Schema, the value MUST conform to the defined type for the Schema Object defined at the same level. For example, if `type` is `string`, then `default` can be `"foo"` but cannot be `1`.
+- default - The value MUST conform to the defined type for the Schema Object defined at the same level. For example, if `type` is `string`, then `default` can be `"foo"` but cannot be `1`. This differs from JSON Schema which only RECOMMENDS the default conforms to the schema.
 
 Alternatively, any time a Schema Object can be used, a [Reference Object](#referenceObject) can be used in its place. This allows referencing definitions instead of defining them inline.
 
-Additional properties defined by the JSON Schema specification that are not mentioned here are strictly unsupported.
-
-Other than the JSON Schema subset fields, the following fields MAY be used for further schema documentation:
+In addition to the JSON Schema fields, the following OpenAPI vocabulary fields MAY be used for further schema documentation:
 
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="schemaNullable"></a>nullable | `boolean` | Allows sending a `null` value for the defined schema. Default value is `false`.
+<a name="schemaNullable"></a>nullable | `boolean` | Allows sending a `null` value for the defined schema. Default value is `false`. Deprecated. Move towards using `"type": "null"` & type arrays , e.g: `"type": ["string", "null"]`.
```

Comment:
+1 for deprecating `nullable`!

Also +1 for changing its default as suggested.

But I don't think we can deprecate it until we allow at least the simple form of type arrays that you proposed in #1389:
> * `type` MUST be a string or a two-element array with unique items
> * if an array, one of the unique items MUST be `"null"`

... or just add support for type arrays, without restricting it to the simplified two-element form. I'm not sure there's really any benefit to that restriction. 

If we can make these changes in 3.1, then the proposed JSON Schema vocabulary can assume that change will already be in effect.

Reply:
@tedepstein "or just add support for type arrays" this PR does that. :)

Reply:
@handrews yeah I need to write a thing in the spec which says "nullable: true" should be treated the same as a null appearing in the array, instead of leaving it as default false. they're certainly not meant to contradict each other.

Reply:
The greater issue here is that the default, as in "if omitted", has a constricting behaviour.
Omitting keywords should not cause additional validation constraints to be imposed, according to JSON Schema principals.